packages/common/src/services/JWPEntitlementService.ts

```diff
@@ -1,11 +1,22 @@
-import InPlayer from '@inplayer-org/inplayer.js';
-import { injectable } from 'inversify';
+import { inject, injectable } from 'inversify';
+
+import JWPAPIService from './integrations/jwp/base/JWPAPIService';
+import type { SignedMediaResponse } from './integrations/jwp/base/types';
 
 @injectable()
 export default class JWPEntitlementService {
+  private readonly apiService;
+
+  constructor(@inject(JWPAPIService) apiService: JWPAPIService) {
+    this.apiService = apiService;
+  }
+
   getJWPMediaToken = async (configId: string = '', mediaId: string) => {
     try {
-      const { data } = await InPlayer.Asset.getSignedMediaToken(configId, mediaId);
+      const data = await this.apiService.get<SignedMediaResponse>(`v2/items/jw-media/token?app_config_id=${configId}&media_id=${mediaId}`, {
```

Comment:
Could be nice to pass query params to "get" in the future as a separate argument. Looks cleaner to me.

Reply:
Done.